src/components/common/ColonyActions/helpers/getEventTitleValues.ts

```diff
@@ -133,6 +138,17 @@ const EVENT_TYPE_MESSAGE_KEYS_MAP: {
     EventTitleMessageKeys.ReputationChangeNumeral,
     EventTitleMessageKeys.Recipient,
   ],
+  [SystemMessagesName.ObjectionFullyStaked]: [
```

Comment:
These could live in their own map, `MOTION_MESSAGE_KEYS_MAP`, or similar

Reply:
@willm30 just to be clear you are thinking we should split EVENT_TYPE_MESSAGE_KEYS_MAP into separate maps for Actions and Motions? and the System messages will live in the motions map?